### PR TITLE
Create lit-element@next-major issue template

### PR DESCRIPTION
**Pull request by arthurevans**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+-->
+
+# Description
+A clear and concise description of what the bug is.
+
+# Lit version
+
+<!-- 
+Run this command from your code or in the console:
+console.log(`lit-element: ${window.litElementVersions} lit-html: ${window.litHtmlVersions}`);
```
Comment (justinfagnani) [inline .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md:21]: we could also include an `npm ls` command here: `npm ls lit-html lit-element`

File: .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+about: Use this template to report an issue with the LitElement PREVIEW release (lit-element@next-major)
+title: '[lit-element] '
+labels: 'lit-next'
+assignees: ''
+
+---
+
+<!--
+Please read our contribution guidelines:
+  https://github.com/Polymer/lit-html/blob/master/CONTRIBUTING.md#filing-issues
```
Comment (justinfagnani) [inline .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md:11]: Do we want to include some text about searching for known issues, like in the new READMEs and CHANGELOGs?

File: .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+-->
+
+# Description
+A clear and concise description of what the bug is.
+
+# Lit version
```
Comment (justinfagnani) [inline .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md:17]: version -> versions?

**Review by justinfagnani — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md (hunk 2 above)
Comment (arthurevans) [reply .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md:11]: I can't find that text in the new READMEs and CHANGELOGs, only in the blog post, FWIW. But will add it here.

Comment (arthurevans) [inline .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md:11]:
```suggestion
  https://github.com/Polymer/lit-html/blob/master/CONTRIBUTING.md#filing-issues
  
Before filing a new issue, please check whether it's been reported already. Issues for the next major versions will have the label lit-next.
```

File: .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md (hunk 3 above)
Comment (arthurevans) [inline .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md:17]:
```suggestion
# Lit versions
```

File: .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md (hunk 1 above)
Comment (arthurevans) [inline .github/ISSUE_TEMPLATE/lit-element-next-bug-report.md:21]:
```suggestion
console.log(`lit-element: ${window.litElementVersions} lit-html: ${window.litHtmlVersions}`);  

Or run the following command on the command line:
npm ls lit-html lit-element
```

**Comment by arthurevans** [comment]
Updated, PTAL

**Review by justinfagnani — APPROVED** [review]

**Pull request merged**